src/utils/snaps.test.ts

```diff
@@ -294,4 +295,36 @@ describe('getLatestSnapVersion', () => {
       'No latest version found for Snap: foo-snap.',
     );
   });
+
+  it('returns latest version checksum', () => {
+    const snap: VerifiedSnap = {
+      versions: {
+        // @ts-expect-error - Technically not a valid version.
+        '1.0.0': {
+          checksum: 'foo',
```

Comment:
there is a constant from test-utils, please use SNAP_SHASUM_1, in case you can add another constant, e.g SNAP_SHASUM_2

Reply:
Will add one more SNAP_SHASUM_2 and apply constant to this test.